runtime/service_integ_test.go

```diff
@@ -316,3 +329,99 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 
 	vmWg.Wait()
 }
+
+func TestStubBlockDevices_Isolated(t *testing.T) {
+	internal.RequiresIsolation(t)
+	const vmID = 0
+
+	ctx := namespaces.WithNamespace(context.Background(), "default")
+
+	client, err := containerd.New(containerdSockPath, containerd.WithDefaultRuntime(firecrackerRuntime))
+	require.NoError(t, err, "unable to create client to containerd service at %s, is containerd running?", containerdSockPath)
+	defer client.Close()
+
+	image, err := client.Pull(ctx, debianDockerImage, containerd.WithPullUnpack, containerd.WithPullSnapshotter(naiveSnapshotterName))
+	require.NoError(t, err, "failed to pull image %s, is the the %s snapshotter running?", debianDockerImage, naiveSnapshotterName)
+
+	rootfsBytes, err := ioutil.ReadFile(defaultVMRootfsPath)
```

Comment:
Can you move this under the comment about immutable rootfs so the code that is related is grouped together?

```go
// TODO once Noah's immutable rootfs change...
rootfsBytes, err := ioutil.ReadFile(defaultVMRootfsPath)
require.NoError(t, err, "failed to read rootfs file")
rootfsPath := fmt.Sprintf("%s.%d", defaultVMRootfsPath, vmID)
err = ioutil.WriteFile(rootfsPath, rootfsBytes, 0600)
require.NoError(t, err, "failed to copy vm rootfs to %s", rootfsPath)
```